modules/dnn/src/dnn_read.cpp

```diff
@@ -51,6 +51,10 @@ Net readNet(const String& _model, const String& _config, const String& _framewor
             std::swap(model, config);
         return readNetFromModelOptimizer(config, model);
     }
+    if (framework == "tensorrt" && (modelExt == "onnx" || modelExt == "trt"))
```

Comment:
please define what you mean under trt file? In case if it's TensorRT engine dump, then it's bad idea. It's not portable and may case unexpected faults.

Reply:
The `trt` file is a cache file obtained by setting the `useCache` of `TrtConfig` when using `readNetFromTensorRT`. Usually this file cannot be reused on other machines with different environments. It is related to the TensorRT version and the compute capability of the GPU.

Reply:
Need to discuss the option. The files are not portable, even CUDA update may  break it. It leads to unpredictable crashes.

Reply:
It may take about 10 mins without this file to convert the `onnx` to `trt` cache file.